scripts/open-coredump.sh

```diff
@@ -343,4 +343,12 @@ Good luck!
 EOF
 fi
 
-exec ${SCYLLA_REPO_PATH}/tools/toolchain/dbuild -it -v $(pwd):/workdir -v ${SCYLLA_REPO_PATH}:/src/scylla -v ${ARTIFACT_DIR}/scylla.package:/opt/scylladb:Z -w /workdir -- bash -l
+TOP_SRCDIR=$(dirname $(dirname $0))
+IMAGE="${SCYLLA_REPO_PATH}/tools/toolchain/image"
+exec ${TOP_SRCDIR}/tools/toolchain/dbuild               \
+    --image "$(<"$IMAGE")"                              \
```

Comment:
i should have passed the content in the `$IMAGE` as the argument to `--image`, not the path to it.